src/features/landing/components/Header/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,95 @@
+import { Drawer, Box, Stack, Button } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { UiButton } from '@/components';
+
+import Bars from '../../../assets/svg/header-drawer/menu-04.svg';
+import CloseImage from '../../../assets/svg/header-drawer/x-close.svg';
+import Logo from '../../../assets/svg/logo/Logo.svg';
+import { navList, socialMedia } from '../dataArray';
+
+import { NavList } from './NavList';
+import { SocialMedia } from './SocialMedia';
+import styles from './styles';
+import { VilnaGmail } from './VilnaGmail';
```

Comment:
what is vilnaGmail?

Reply:
why did you add the name 'gmail'? this is strange because in our email we don't have gmail or something like this, need to fix it